docs/get-started/basic-content.qmd

```diff
@@ -0,0 +1,173 @@
+---
+title: Configuring content
+jupyter:
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+Individual content entries (e.g. a function to be documented) can also be customized.
+For example, if you are documenting a python class, you may want to include or exclude
+documentation on specific methods on that class.
+
+Specify content options by setting `name: <content name>`, along with any additional options.
+
+For example, below is a piece of content, MdRenderer, specified without options.
+
+```yaml
+contents:
+  - MdRenderer
+```
+
+We set it to only document its render method, by setting `name: MdRenderer` followed by the
+`members` option.
+
+```yaml
+contents:
+  - name: MdRenderer
+    members:
+      - render
+```
+
+Behind the scenes, quartodoc represents each content entry as a [](`quartodoc.Auto`) element, since it specifies how to automatically document some object.
+
+## Looking up objects
+
+Finding python objects to document involves two pieces of configuration:
+
+* the package name.
+* a list of objects for content.
+
+Note that quartodoc can look up anything---whether functions, modules, classes, attributes, or methods.
```

Comment:
Can you give a little context on where the package is being imported from? What version would I expect the docs package to be, etc?

Reply:
I agree this is useful and am stumped on where to put it, so teed it up for next time someone runs through quartodoc from scratch 😅

https://github.com/machow/quartodoc/issues/123